app/models/employee.rb

```diff
@@ -36,6 +36,10 @@ def self.current_or_future
           ' or start_date > :today', today: Time.zone.today
   end
 
+  def self.past_or_future
```

Comment:
I was testing it out to confirm it fixes both salaries and graphs and discovered this was missing

![Screen Shot 2020-12-30 at 9 41 14 AM](https://user-images.githubusercontent.com/6103923/103358340-5dcfc200-4a83-11eb-950c-49ee631d784e.png)

Now this works.

@aterrype